lib/consoleInOut.rb

```diff
@@ -1,6 +1,23 @@
-class Input
+class ConsoleInOut
+  attr_reader :input, :output
+
+  def initialize(input, output)
+    @input = input
+    @output = output
+  end
+
+  def write(to_write)
+    if @output.kind_of?(Array)
```

Comment:
What do we want to be the responsibility of the Console and what is about displaying things.